etcd/src/test/java/org/jclouds/etcd/features/KeysApiLiveTest.java

```diff
@@ -94,6 +94,40 @@ public void testDeleteNonExistentKey() {
       assertNull(deletedKey);
    }
 
+   @Test
+   public void testWaitKey() {
+      String localKey = randomString();
+      String localValue = randomString();
+      Key createdKey = api().createKey(localKey, localValue, 3);
+      assertNotNull(createdKey);
+      assertNotNull(createdKey.node().expiration());
+      assertTrue(createdKey.node().ttl() == 3);
+
+      createdKey = api().waitKey(localKey);
```

Comment:
@nacx does jclouds have any default timeout that trips should the call not return in a specified amount of time? And if so is there a way to disable it for these "wait" methods of the KeysApi?